### ISSUE TITLE
Forward OAuth Identity not working

### ISSUE THREAD

**Issue by anas-aso** [humio/humio2grafana#85]
**Describe the bug**
I get `Method Not Allowed` error when I try to configure the datasource with `Forward OAuth Identity` (see Humio docs [recommendation](https://docs.humio.com/docs/integrations/grafana/#notes-on-authentication))

**To Reproduce**
<img width="948" alt="Screenshot 2021-04-30 at 01 56 41" src="https://user-images.githubusercontent.com/16538065/116632462-90b59400-a957-11eb-9511-003163337533.png">

**Expected behavior**
A working datasource.

**Screenshots**
see above

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser : chrome
 - Version :  90.0.4430.93 (Official Build) (x86_64)

**Additional context**
The datasource works if I use an API token instead.

**Comment by AlexanderBrandborg** [comment]
Hi @anas-aso.
Thanks for notifying us about this, I'm able to reproduce this issue locally.
I trust you can still use the plugin fine, even when you have to auth using the token? 
I'll note you when I have a solution ready :- )

**Comment by anas-aso** [comment]
>  I trust you can still use the plugin fine, even when you have to auth using the token?

@AlexanderBrandborg yes, the plugin works if I use an API token, but this means that all Grafana users get the same permissions level as that API token which is not what I need.

**Comment by AlexanderBrandborg** [comment]
@anas-aso ,
My apologies,  there has been a misunderstanding on my part.  "Forward OAuth" identify in Grafana seems to mean that you're forwarding your Grafana Oauth token to Humio for authentication, which is not a valid manner of authenticating against Humio. I was of the understanding that it would forward the OAuth token granted by Humio not Grafana.

So login with OAuth is not something we support right now, and I will update the docs to reflect this.

However, am I right in assuming that you want to authenticate using the OAuth token granted by Humio correct? That I should be able to include in the plugin for its next version. :)

**Comment by anas-aso** [comment]
@AlexanderBrandborg I can already authenticate using my personal Humio token.

What I am interested in is this :
> This will allow Grafana users to interact with the Humio data source as their own Humio user.

source : https://docs.humio.com/docs/integrations/grafana/#notes-on-authentication

My use case is this : 
1. I have a user that authenticate to Grafana using Google OAuth
2. That same user is added to Humio and authenticate against Humio via Google OAuth as well
3. When that user builds graphs using Humio datasource, I want him/her to only access what has been authorized on Humio RBAC.

**Comment by AlexanderBrandborg** [comment]
@anas-aso thanks for the detail, I'll be looking into this for the next plugin version :)

**Comment by anas-aso** [comment]
@AlexanderBrandborg any updates on this ?
